libraries/AP_InertialSensor/AP_InertialSensor_SITL.cpp

```diff
@@ -119,9 +130,17 @@ void AP_InertialSensor_SITL::generate_gyro(uint8_t instance)
     float q = radians(sitl->state.pitchRate) + gyro_drift();
     float r = radians(sitl->state.yawRate) + gyro_drift();
 
-    p += gyro_noise * rand_float();
-    q += gyro_noise * rand_float();
-    r += gyro_noise * rand_float();
+    const Vector3f &vibe_freq = sitl->vibe_freq;
```

Comment:
why is vibe_freq a member variable in SITL.h and a local variable reference?

Reply:
it is a convenient alias, to make the code neater. It converts from AP_Vector3f to Vector3f, and makes the following code easier to read